rfcs/text/0017-remove-log-original.md

```diff
@@ -16,16 +16,18 @@ The removal of `log.original` will be considered a breaking change since the fie
 
 | Field Set | Field(s) |
 | --------- | -------- |
-| [`log`](0000/log.yml) | `log.original` |
-| [`event`](0000/event.yml) | `event.original` |
+| [`log`](0017/log.yml) | `log.original` |
```

Comment:
Including field definition files when adding or changing fields has proven useful in a couple of ways:

* The fields can be used either with the `--include` flag to test out the field changes
* Sometimes, the proposed field definitions get a little long, especially for brand new fieldsets. Separating the field definitions from the markdown keeps everything a bit tidier.

The way that custom files are merged with the current ECS fields, even if you remove a current ECS field it'll still be present in all the outputs. So it's not a problem to include these field definitions here, but unfortunately, we can't use them with the `--include` flag and get the expected result.

I do think we could consider having a consistent way to mark a field as `deprecated` in the ECS docs until we're ready to entirely remove the field. Looking at the Elastic docs repo, there is an asciidoc admonition available for `deprecated` that we might be able to leverage somehow: https://github.com/elastic/docs#old-section.

We have something similar for the `beta` attribute: https://github.com/elastic/ecs/blob/master/scripts/templates/field_details.j2#L13-L18